src/components/searchInput/_searchInput.scss

```diff
@@ -0,0 +1,39 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@500&display=swap');
+
+$font-nunito: 'Nunito', sans-serif;
+@mixin nunito-medium {
+  font-family: $font-nunito;
+  font-weight: 500;
+}
+
+.search-container {
+    width: 350px;
```

Comment:
It may be better not to give a fixed width to the container, 'max-width' or 'min-width' will give more flexibility to the element

Reply:
Done!